### PR TITLE
Remove additional projects no longer used with CXF 2.6.2 removal

### PR DESCRIPTION
**Pull request by jhanders34**
- Remove projects used with CXF 2.6.2 for wssecurity
- Remove 2.3 version of the JAXB and JAXWS API
- Remove 3.2 version of wsat.cxf.utils.3.2 project
- Update projects' buildpath to remove references to removed projects
- Update use of jaxws23 to be jaxws from rename of project.
- Update to use 3.1.1 version of neethi with cxf 3.2 for jaxwsClient
- Remove features that aren't used any more.
- Update features to remove reference to wsat.cxf.utils.3.2
- Update code to not use wsat.cxf.utils.3.2, but instead to call the cxf code directly now that there is only one cxf version.
- Update saml.sso tests to use JDK Base64 instead of xmltooling
- Remove removed artifacts from dependency maven files.

### CODE REVIEW

**Comment by jhanders34** [comment]
#build

**Comment by jhanders34** [comment]
#build

File: dev/cnf/oss_ibm.maven [hunk 1]
```diff
@@ -28,19 +28,8 @@ com.ibm.ws.logstash:logstash:5.5.0
...
 com.ibm.ws.org.apache.commons:commons-compress:1.16.1
-com.ibm.ws.org.apache.cxf:cxf-api:2.6.2.ibm-s20190122-2227
-com.ibm.ws.org.apache.cxf:cxf-rt-bindings-soap:2.6.2.ibm-s20180529-1900
-com.ibm.ws.org.apache.cxf:cxf-rt-bindings-xml:2.6.2.ibm-s20170216-1739
-com.ibm.ws.org.apache.cxf:cxf-rt-core:2.6.2.ibm-s20180529-1900
-com.ibm.ws.org.apache.cxf:cxf-rt-databinding-jaxb:2.6.2.ibm-s20170216-1739
-com.ibm.ws.org.apache.cxf:cxf-rt-features-clustering:3.2.0.20170815-120200-511
-com.ibm.ws.org.apache.cxf:cxf-rt-frontend-jaxws:2.6.2.ibm-s20180914-1510
-com.ibm.ws.org.apache.cxf:cxf-rt-frontend-simple:2.6.2.ibm-s20170216-1739
-com.ibm.ws.org.apache.cxf:cxf-rt-management:2.6.2.ibm-s20170216-1739
```
Comment (neuwerk) [inline dev/cnf/oss_ibm.maven:39]: Glad you got these too!

File: dev/cnf/dependabot/check_this_in_if_it_changes/pom.xml [hunk 2]
```diff
@@ -1191,11 +1181,6 @@
       <artifactId>jaxb-api</artifactId>
       <version>2.3.0</version>
```
Comment (neuwerk) [inline dev/cnf/dependabot/check_this_in_if_it_changes/pom.xml:1182]: Can we drop JAX-B 2.3 (API and rest of the impl) now as well?

File: dev/cnf/oss_dependencies.maven [hunk 3]
```diff
@@ -59,9 +59,7 @@ com.sun.xml.bind.external:rngom:3.0.1
 com.sun.xml.bind:jaxb-impl:2.2.10
 com.sun.xml.bind:jaxb-impl:2.3.0.1
```
Comment (neuwerk) [inline dev/cnf/oss_dependencies.maven:60]: Could be removed if we are no longer needing the JAX-B 2.3 impl, depending on the answer to the question above.

File: dev/cnf/oss_dependencies.maven [hunk 4]
```diff
@@ -234,7 +232,6 @@ javax.ws.rs:javax.ws.rs-api:2.1-m09
 javax.xml.bind:jaxb-api:2.2.12-b140109.1041
 javax.xml.bind:jaxb-api:2.2.7
 javax.xml.bind:jaxb-api:2.3.0
```
Comment (neuwerk) [inline dev/cnf/oss_dependencies.maven:234]: Could be removed if we are no longer needing the JAX-B 2.3 API, depending on the answer to the question above.

File: dev/cnf/dependabot/check_this_in_if_it_changes/pom.xml (hunk 2 above)
Comment (jhanders34) [reply dev/cnf/dependabot/check_this_in_if_it_changes/pom.xml:1182]: It is still used by the build infrastructure so I won't going to remove it.

**Review by arunavemulapalli — APPROVED** [review]
ws-security, saml and related changes look good to me.
you are already aware and mentioned that we need WS-CD-Open changes to go along with these (where we are still referencing the deleted projects and needs to be updated).

thank you

**Comment by posmith** [comment]
L2 message review completed

**Comment by jhanders34** [comment]
#libby

**Pull request merged**